templates/bind9/0/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+bind9:
+  image: digitallumberjack/docker-bind9:v1.0.0
+  ports: 
+   - 53:53/tcp
```

Comment:
I realize these are the obvious choices :) But we get a lot of community feedback to make the public ports configurable via the questions. Can you make these configurable and set the defaults to 53?